Formula/rdkit.rb

```diff
@@ -34,7 +35,13 @@ def install
     python_executable = Formula["python@3.9"].opt_bin/"python3"
     py3ver = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
     py3prefix = Formula["python@3.9"].opt_frameworks/"Python.framework/Versions/#{py3ver}"
+    # py3prefix on Linux is different from one on macOS 
```

Comment:
```suggestion
    # py3prefix on Linux is different from one on macOS
```
I think we can omit this comment, actually.